integration/oasis/solution/solution.go

```diff
@@ -24,3 +29,38 @@ type Location struct {
 	Lat float64
 	Lon float64
 }
+
+// Convert2ExternalSolution convert internal solution format to external format defined in pkg/api/oasis/response
+func (sol *Solution) Convert2ExternalSolution() *oasis.Solution {
```

Comment:
Where is the "oasis.Solution"? I went through codes but didn't find it.

Reply:
https://github.com/Telenav/osrm-backend/blob/0cf096845aa6067be8625ee4abb671af56c5abc5/integration/pkg/api/oasis/response.go#L13

Reply:
Oh! Got it! Currently it's ok to me. Let's discuss it later, same question on osrm api/impl too.